apps/designer/package.json

```diff
@@ -74,15 +74,7 @@
   "devDependencies": {
     "@babel/core": "^7.20.2",
     "@remix-run/dev": "^1.6.4",
-    "@storybook/addon-actions": "^6.5.9",
```

Comment:
I couldn't find any reference to this one (`@storybook/addon-actions`) being used anywhere in the codebase - so I didn't include it in the common config anyhow (nor in any of the ones using the common config). Should I bring it back?